src/starter.sh

```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
-ruby /sectools/src/main.rb & \
-bash /sectools/arachni/bin/arachni_rest_server
```

Comment:
Arachni actually requires the starter script to start the arachni api server.
Without it the arachni api doesnt get started, meaning that no scans can get started.